### DISCUSSION TITLE
2023/01/31/5e91c1995fd8/

### DISCUSSION THREAD

**Discussion by giscus** [bijinfeng/bijinfeng.github.io#22] [Announcements]
# 2023/01/31/5e91c1995fd8/

背景我的博客都是用 Hexo 在本地写完，提交个 commit 到 Github，触发 Github Action 自动部署到 Github Page 上。除了用 Vscode 写 Markdown 有点难受外，其它地方还好，至于为什么一定要用 Hexo + Github 的组合，白嫖 Github 的服务器它不香吗？ 但是，最近体验了下语雀，发现他的文本编辑器还是很不错的，写文档很是丝滑，正好在

https://blog.kebai.club/2023/01/31/5e91c1995fd8/

**Comment by bijinfeng** [comment]
测试评论
